redi-connect-infrastructure/README.md

```diff
@@ -2,91 +2,123 @@
 
 In this project we keep all the infrastructure redi-connect needs to run. We do that writing infrastructure as code using [terraform](https://www.terraform.io/).
 
-## Prerequisites
-
-The first time one executes terraform on an azure subscription they would have to create an azure container where they can store terraform state.
+## Prerequisites & Getting started
 
+Before using using Terraform on an Azure subscription, an Azure container is required to store Terraform state.
 
 This is the list of prerequisites required for the `connect` ci/cd pipeline:
 
-1. Before you run `terraform init` - Create the backend for azure.
-
-    resource:  https://docs.microsoft.com/en-us/azure/developer/terraform/store-state-in-azure-storage
-    
-    We do this using terraform. The tf code to provision the backend is in the `az-remote-backend-*` files.
-    When you `terraform aply` for the first time the backend will be provisioned for you in azure.
-    Also you will see the outputs, something like:
-    
-    ```
-    storage_account_name: tstate8694
-    container_name: tstate
-    access_key: ****
-    
-    ``` 
-    You will have to copy the `storage_account_name: terraformstateXXXX` and paste them in the `_versions.tf` file like so
-    ```
-     backend "azurerm" {
-       storage_account_name  = "terraformstateXXXX"
-        .
-        .
-        .
-     }
-    ```
-    
-    Then you can run `terraform init`
-    
+1. Create the backend for Azure.
+
+   This is step is needed only the very first time _before_ you run `terraform init`. This is because when you are under a completely new Azure subscription you dont have a storagr container to store the Terraform state yet.
```

Comment:
Typo,`storagr` => `storage`

Reply:
```suggestion
   This is step is needed only the very first time _before_ you run `terraform init`. This is because when you are under a completely new Azure subscription you dont have a storage container to store the Terraform state yet.
```

Hey @ericbolikowski you can press this (see image) to make suggestions. This way we can commit the directly here. This works pretty well when suggestions are that small as this one ☝️ 
![image](https://user-images.githubusercontent.com/4181674/109294752-837c0c00-7825-11eb-8cc4-dbb19cec95f0.png)

Reply:
Ah, neat! 🙌 I'd never seen this before.